.github/workflows/linter.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Lint Code Base
+
+on:
```

Comment:
This PR does more than it says in the description. I think adding the linter for example would have made a great separate PR.